Document/0x06d-Testing-Data-Storage.md

```diff
@@ -103,7 +103,37 @@ Dumping the iOS keychain...
 Created                    Accessible                      ACL    Type      Account                    Service                                                        Data
 -------------------------  ------------------------------  -----  --------  -------------------------  -------------------------------------------------------------  ------------------------------------
 2020-02-11 13:26:52 +0000  WhenUnlocked                    None   Password  keychainValue              com.highaltitudehacks.DVIAswiftv2.develop                      mysecretpass123
-```  
+```
+
+The dedicated script referenced in the [tools](#tools) section can be used to find common entries between two Keychain dump.
+
+Use it like this:
+
+1. Install the application and use it (inital install).
+2. Dump the Keychain using `objection` to a json file.
```

Comment:
```suggestion
2. Dump the Keychain using `objection` to `DumpKeyChainAfterInitialInstall,json`
```

Reply:
I have also fixed the typo `DumpKeyChainAfterInitialInstall,json` by `DumpKeyChainAfterInitialInstall.json`